corehq/apps/cloudcare/templates/preview_app/base.html

```diff
@@ -9,6 +9,10 @@
           rel="stylesheet"
           media="all"
           href="{% static 'preview_app/less/preview_app.less' %}"/>
+      <link type="text/less"
```

Comment:
I'm surprised this moved - this file is only used by app preview, but `dependencies.html` is included by both app preview and web apps (and edit forms, a legacy feature that depends on web apps). Do the styles still show up in web apps?

Reply:
Yep, the styles are getting applied to tables.

I thought since we seeing tables in the app preview, it would be nice to move it there.

<img width="1152" alt="Screenshot 2020-09-18 at 5 34 04 PM" src="https://user-images.githubusercontent.com/7694243/93596002-54d84a80-f9d6-11ea-8503-692fcbeddd0c.png">

Reply:
They definitely look good in app preview!  Can you confirm they're also still showing up in web apps?

Reply:
Yep. They are.
I can move the import back to `dependencies.html` if that would be a preferred way to do it?